### ISSUE TITLE
SteamMultiplayerPeer: lobby_created signal isn't firinig

### ISSUE THREAD

**Issue by Fifut** [GodotSteam/MultiplayerPeer#1]
**Describe the bug**
`lobby_created` signal isn't firing after calling (SteamMultiplayerPeer) `create_lobby(SteamMultiplayerPeer.LOBBY_TYPE_PUBLIC)`
Only `lobby_joined` signal is firing
But work with `Steam.createLobby(Steam.LOBBY_TYPE_PUBLIC, 8)`

**To Reproduce**
Init steam
Connect `lobby_created`  signal
Create a new `SteamMultiplayerPeer`
call `create_lobby(SteamMultiplayerPeer.LOBBY_TYPE_PUBLIC)` on this peer

**Desktop (please complete the following information):**
 - OS: Windows 11 Professionnel
 - Version 23H2

**Version of GodotSteam:**
Godot 4.3 - Steamworks 1.60 - GodotSteam MultiplayerPeer 4.10

**Additional context**
I didn't forget `Steam.run_callbacks()` in `_process()`

**Comment by Gramps** [comment]
Necro-issue!

@NiclasEriksen Did you ever solve this?  I noticed you are using the `Steam` object but you want to be using the `SteamMultiplayerPeer` one instead.  Also, are you using `run_callbacks`?  Are you getting any other signals?

@Fifut Since this has been revived, have you figured it out?  You should also be connecting to the `SteamMultiplayerPeer` object for those signals, not `Steam`; I think.  Part of why I don't like this MultiplayerPeer system is the mixing of signals between two objects and the lack of clarity over what should or shouldn't be emitting.

I would assume they should be linked to `SteamMultiplayerPeer` but I could be wrong as I didn't work on this at all.

**Comment by Fifut** [comment]
@Gramps No, I haven’t found the solution yet. I checked the source code and didn’t see any errors, but my C++ level isn’t very high.

In the meantime, I found a workaround by calling the `lobby_created `function from within the `lobby_joined `function if the peer ID is the server:

```
func lobby_joined(lobby_id, _permissions, _locked, _response):
    if _peer.get_unique_id() == 1:
        lobby_created(1, lobby_id)
```
I just tried connecting the signals to the SteamMultiplayerPeer object instead of Steam, and it works!
Thanks for the feedback.
